src/app/modules/round/round.controller.js

```diff
@@ -311,8 +311,17 @@ angular
         if (playerScore > vm.player.bestRoundScore) {
           PlayerStats.updateBestRound(vm.player, playerScore);
         }
-        var nextRound = currentRoundPlayedID.length < SelectedPlayers.getAll().length ? vm.round : vm.round + 1;
-        $location.path('summary/round/' + nextRound + '/player/' + Match.getNextPlayer(nextRound).id);
+
+        var nextRound;
+        if (currentRoundPlayedID.length < SelectedPlayers.getAll().length) {
```

Comment:
Ammettendo che sia tutto corretto lo riscriverei come
```
var nextRound = currentRoundPlayedID.length < SelectedPlayers.getAll().length ? vm.round : vm.round + 1;
Match.setCurrentPlayer(Match.getNextPlayer(nextRound));
```
`Match.getNextPlayer(nextRound)` non ritorna 0 se è un nuovo round?

Reply:
`Match.getNextPlayer(nextRound)` ritorna, per il nuovo round, il player successivo in fila, quindi varia in base agli skip che sono stati fatti nel round precedente

Reply:
che intendi con "varia in base agli skip che sono stati fatti nel round precedente"?

Reply:
Intendo dire che se su tre giocatori, l'ultimo a giocare è il secondo, nel nuovo round il metodo `Match.getNextPlayer(nextRound)` restituirà il terzo giocatore, invece del primo